experimentA/idr0051-experimentA-assays.txt

```diff
@@ -0,0 +1,6 @@
+Dataset Name	Image File	Characteristics [Organism]	Term Source 1 REF	Term Source 1 Accession	Characteristics [Organism Part]	Term Source 2 REF	Term Source 2 Accession	Characteristics [Ecotype]	Term Source 3 REF	Term Source 3 Accession	Characteristics [Genotype]	Characteristics [Developmental Stage]	Protocol REF 1	Protocol REF 2	Protocol REF 3	Channels	Image File Type 	Comment [Image File Type]	Comment [Image File]
+Sample 3	180712_H2B_22ss_Courtney1_20180712-163837_p00_c00_preview.pattern	Danio rerio	NCBITaxon	7955	Tailbud	ZFA	ZFA_0000077	T�pfel long fin	EFO	EFO_0003332	H2B-GFP	22 Somite Stage	Lightsheet Protocol	Online tracking light sheet timelapse protocol	TGMM tracking	GFP	TIFF	preview, registered	Time point 0 - 78
```

Comment:
This file contains a few symbols like `T�pfel` - does this work correctly at annotation time?

Reply:
I just manually replaced that with 'u' in the annotation.csv.